turbo/jsonrpc/trace_adhoc.go

```diff
@@ -1356,6 +1357,8 @@ func (api *TraceAPIImpl) doCallMany(ctx context.Context, dbtx kv.Tx, msgs []type
 		}
 	}
 
+	ibs.Reset()
```

Comment:
feels weird that we return `ibs` object which was reset. who needs an empty object.